src/constants/translations/en/my-courses-page.json

```diff
@@ -38,5 +38,10 @@
     "applyFilters": "Apply filters",
     "categoryLabel": "Category",
     "subjectLabel": "Subject"
+  },
+  "errorMessages": {
```

Comment:
already have these errors in offer-page.json, maybe make sense to move them for example to common.js

Reply:
> already have these errors in offer-page.json, maybe make sense to move them for example to common.json

moved